pubnub/lib/src/dx/_endpoints/history.dart

```diff
@@ -154,25 +154,46 @@ class BatchHistoryResultEntry {
   /// Otherwise, it will be `null`.
   Map<String, dynamic>? meta;
 
+  /// Error message. this will contain information if message decryption is failed
+  /// for given `message`.
+  String? error;
```

Comment:
Don't you have custom class/type that describes an error? If there's no such a class, let's leave it as it is.

Reply:
Fixed